### ISSUE TITLE
Add a "settings saved" confirmation in admin view

### ISSUE THREAD

**Issue by evan-lauer** [DigitalCarleton/AccessibilityPlus#12]
Might be nice to have a little green banner pop up when settings are saved to confirm that it worked correctly. Is that possible?

**Comment by apjmason** [comment]
Yes! Omeka uses the Zend flashMessenger helper to do this. The 'success' category will be green.

The [developer docs](https://omeka.readthedocs.io/en/latest/Reference/controllers/helpers/FlashMessenger.html) explain the methods, but the easiest way to see how it works is to search one of the omeka plugin repos to find an example of it in action, e.g. [ExhibitBuilder 'success' example](https://github.com/omeka/plugin-ExhibitBuilder/search?q=flashmessenger)

**Comment by evan-lauer** [comment]
Ok cool -- assigning myself for this!
